### ISSUE TITLE
Wrong video playing time

### ISSUE THREAD

**Issue by tudosxxx** [sbis04/video_trimmer#192]
The video over 14s is not played
`final trimAreaDuration = Duration(
            milliseconds: (maxVideoLength.inMilliseconds +
                ((paddingFraction * maxVideoLength.inMilliseconds) * 2)
                    .toInt()));`
[log] Total Video Length: 0:00:16.280000
[log] Max Video Length: 0:00:10.000000
[log] Padding Fraction: 0.2
[log] Trim Area Duration: 0:00:14.000000

16s video, only 14s clipping area

**Comment by tudosxxx** [comment]
suggest：TrimEditorPainter  and  showtime  Format Customization

**Comment by tudosxxx** [comment]
https://user-images.githubusercontent.com/52568890/210762484-e0d985b7-60ec-4fe8-994a-1a1089139eae.mp4

**Comment by tudosxxx** [comment]
if maxVideoLength = 5     trimAreaDuration  = 7    16S video has 7s to edit

**Comment by tudosxxx** [comment]
I run your demo with the same problem. The frame list at the bottom cannot be dragged

**Comment by ruhulmath08** [comment]
Yes, I also fetch the same issue. The frame list at the bottom cannot be dragged.
